packages/packages/commonwell-sdk/src/models/link.ts

```diff
@@ -12,9 +12,16 @@ export const linkSchema = z.object({
 export type Link = z.infer<typeof linkSchema>;
 
 export const networkLinkSchema = z.object({
-  _links: linkSchema.optional().nullable(),
+  _links: z
+    .object({
+      self: linkSchema.optional().nullable(),
+      upgrade: linkSchema.optional().nullable(),
+      downgrade: linkSchema.optional().nullable(),
+    })
+    .optional()
+    .nullable(),
```

Comment:
not this line, but the one below - assuranceLevel is supposed to be an int?

also, should we make it an enum with values 0 - 4 ? idea:

```
enum LOLA {
  level_0 = 0,
  level_1 = 1,
  leve_2 = 2,
  level_3 = 3,
  level_4 = 4,
}
```

Reply:
So its supposed to be an int but in the payload its a string